plotting/comparison.py

```diff
@@ -95,7 +95,7 @@
     mergemap[c] = comparelist
 collated = collate(output, mergemap)
 ### style settings
-if "Run" in args.ref:
+if args.ref.count("Run") > 1:
```

Comment:
Ah, I did not consider that this would happen if we add the campaign name `BTV_Run3_2022_Comm_v1` into the dataset names of the json files. Besides being unintuitive, this code will fail if we choose a different campaign name in the future that doesn't contain `Run`.

I think, the better alternative would be not to add the suffix `BTV_Run3_2022_Comm_v1` to the dataset names inside the json files, but rather rename the json files themselves as follows:
```
MC_BTV_Run3_2022_Comm_v1.json
data_<FLAVOR>_BTV_Run3_2022_Comm_v1.json
```
with `<FLAVOR>` = mu, mumu, or emu.
By doing so, runner.py will create `.coffea` files with the name tag `BTV_Run3_2022_Comm_v1`, and should be therefore clearly identifiable. At plotting label, `--ext=BTV_Run3_2022_Comm_v1` would be a good option to make clear which campaign has been used for the plots.

Reply:
Sounds like a good idea, I will modify accordingly